### PR TITLE
adding civil orchestrator service and relevant changes

### PR DESCRIPTION
**Pull request by vasudevganesanhmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by mounikahmcts — DISMISSED** [review]

**Review by mounikahmcts — DISMISSED** [review]

File: apps/civil/civil-orchestrator-service/perftest.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  name: civil-orchestrator-service
+  namespace: civil
+spec:
+  values:
+    java:
+      keyVaults:
+        civil:
+          resourceGroup: civil
+          secrets:
+            - civil-idam-client-secret
```
Comment (adusumillipraveen) [inline apps/civil/civil-orchestrator-service/perftest.yaml:13]: Also, any reason why you are overriding secrets in every environment ?

File: apps/civil/base/kustomization.yaml [hunk 2]
```diff
@@ -5,5 +5,6 @@ resources:
   - ../identity/identity.yaml
   - ../civil-service/civil-service.yaml
   - ../civil-general-applications/civil-general-applications.yaml
+  - ../civil-orchestrator-service/civil-orchestrator-service.yaml
```
[the remark this answers is not in the document]
Comment (vasudevganesanhmcts) [reply apps/civil/base/kustomization.yaml:8]: sure i have corrected in my latest commit

File: apps/civil/civil-orchestrator-service/perftest.yaml (hunk 1 above)
Comment (vasudevganesanhmcts) [reply apps/civil/civil-orchestrator-service/perftest.yaml]: it was added to get access to civil vault but i have removed it as initially we may not need it. .have pushed it

File: apps/civil/base/kustomization.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (adusumillipraveen) [reply apps/civil/base/kustomization.yaml:8]: adding here, adds to all envs. You need to add it to each env base and remove here https://github.com/hmcts/cnp-flux-config/blob/master/docs/app-deployment-v2.md#add-application-to-only-one-environment

[the remark this answers is not in the document]
Comment (vasudevganesanhmcts) [reply apps/civil/base/kustomization.yaml]: thanks ..i have just pushed in the changes required

**Review by adusumillipraveen — APPROVED** [review]

**Review by mounikahmcts — APPROVED** [review]

**Pull request merged**